src/Symfony/Component/AssetMapper/Command/ImportMapInstallCommand.php

```diff
@@ -11,12 +11,14 @@
 
 namespace Symfony\Component\AssetMapper\Command;
 
-use Symfony\Component\AssetMapper\ImportMap\ImportMapManager;
+use Symfony\Component\AssetMapper\Tests\ImportMap\RemotePackageDownloader;
```

Comment:
A command must not reference a class in the `Tests` namespace. this namespace is not shipped when installed packages from dist